build_projects/shared-build-targets-utils/VersionRepoUpdater.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.DotNet.Cli.Build
     public class VersionRepoUpdater
     {
         private static Regex s_nugetFileRegex = new Regex("^(.*?)\\.(([0-9]+\\.)?[0-9]+\\.[0-9]+(-([A-z0-9-]+))?)\\.nupkg$");
-
+        private static string s_baseGitHubPath = $"/build-info/dotnet/cli";
```

Comment:
If that is the case then we should move this out into it's own package and consume it that way. Keeping a source file sync'd across multiple repos without changes is not a viable long-term solution; there is no way to enforce that and keep it synchronized